AElf.Kernel.Tests/TransactionSchedulerTest.cs

```diff
@@ -72,58 +73,89 @@ private static ITransaction CreateTransaction(byte b)
                     int index = random.Next(hashCount);
                     if(index>=hashCount) continue;
                     var h = hashList.ElementAt(index);
-
                     if (!pending.Keys.Contains(h)) pending[h] = new List<ITransaction>();
-                    pending[h].Add(t);
-                } while (random.Next(hashCount) < hashCount / 2);
+                    if(!pending[h].Contains(t)) pending[h].Add(t);
+                } 
+                while (random.Next(hashCount) < hashCount / 2);
             }
 
             return pending;
         }
-        
-        
-        [Fact]
-        public void ScheduleTest()
-        {
-            var h1 = CreateHash(1);
-            var t1 = CreateTransaction(65);
 
+        public void AsyncExecuteTest()
+        {
+            
             Dictionary<IHash, List<ITransaction>> pending = new Dictionary<IHash, List<ITransaction>>();
-            pending[h1]=new List<ITransaction>();
-            pending[h1].Add(t1);
+            
             var transactionExecutingManager = new TransactionExecutingManager { Pending = pending };
-            transactionExecutingManager.Schedule();
-            Assert.Equal(1, Worker.ExecutePlan.Count);
 
-            Worker.ExecutePlan = new Dictionary<IHash, int>();
-            int hashCount = 1;
-            int txCount = 1;
+            // 5 hash and 10 tx
+            int hashCount = 5;
+            int txCount = 10;
+            
             pending = CreatePending(hashCount, txCount);
-            transactionExecutingManager.Pending = pending ;
+            transactionExecutingManager.Pending = pending;
+            
+            Stopwatch stopwatch = new Stopwatch();
+            stopwatch.Start();
```

Comment:
You don't have to use stop watch, C# has some performance test tools, choose one

Reply:
all right